### PR TITLE
Add the form 

### PR DESCRIPTION
**Pull request by jorgegoco**
Add the form part for the mobile skeleton project of Template number 3 from Figma.

### CODE REVIEW

**Review by codepantha — CHANGES_REQUESTED** [review]
Hi @jorgegoco 👋, 

## STATUS: REQUESTED CHANGES ♻️

Good job so far 👏 
There are some issues that you still need to work on to go to the next project but you are almost there!



### Highlights 🎯 ⭐ 
- Beautiful UI ✔️ 
- Linter checks ✔️ 
- Meaningful commit messages ✔️ 



## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jorgegoco** [comment]
Sorry, I can't see where are the required changes. Please let me know to keep on working on it. Thank you.

File: head_mobile_portfolio.html [hunk 1]
```diff
@@ -36,120 +36,143 @@ <h2 class="recent-works">My Recent Works</h2>
...
+			get
+			in touch.</h2>
+		<form class="form" action="https://formspree.io/f/xdobpwbj" method="post">
+			<input class="input-text" name="name" type="text" placeholder="Full name" maxlength="30" required>
+			<input class="input-text" name="email" type="email" placeholder="Email address" required>
+			<textarea class="input-textarea" name="text" maxlength="500" placeholder="Enter text here"
+				required></textarea>
+			<button class="input-button">Get in touch</button>
+		</form>
+		<img class="ilustration-contact-form" src="./images/illustration-contact-form.png" alt="">
```
Comment (codepantha) [inline head_mobile_portfolio.html:165]: - [ ] Please add button interactions such as hover and active to the submit button as shown in the figma design below
image

- [ ] Kindly remove the border around your button to make it look more like the figma design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
Sorry about that, 

Here are the required changes

## Required Changes ♻️
_Check the comments under the review._

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: head_mobile_portfolio.html (hunk 1 above)
Comment (jorgegoco) [reply head_mobile_portfolio.html:165]: Ok. Thanks.

**Review by sja-thedude — APPROVED** [review]
## Project Approved 👍 

Hi @jorgegoco, This is @sja-thedude your code reviewer.

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

giphy

Cheers and Happy coding!👏👏👏

# Optional: This was a pair programming project and there were only commits from your side and not your partner. The previous reviewer missed it that's the reason I am approving it. Kindly remember next time to submit pair programming projects with commits from both the members of the group.

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by jorgegoco** [comment]
Hi.
We already did pair programming but with me as a driver, all the time. You are right, we didn't fully accomplish it in the right way. Thank you for the approval and for kindly suggest me to do it in the right way the next time.

**Pull request merged**